collector/container/scripts/bootstrap.sh

```diff
@@ -71,6 +71,12 @@ function main() {
     # Export SNI_HOSTNAME and default it to sensor.stackrox
     export SNI_HOSTNAME="${SNI_HOSTNAME:-sensor.stackrox}"
 
+    # Get collector and module versions from files and set corresponding environment variables
+    collector_version="$(cat /COLLECTOR_VERSION)"
```

Comment:
I would prefer that we do not create a new dependency in this bootstrap script on COLLECTOR_VERSION AND MODULE_VERSION files.

Reply:
That is my preference as well, but I had difficulty setting COLLECTOR_VERSION and MODULE_VERSION and this is what worked. collector/container/Dockerfile is split into two phases and those two environment variables can only be determined in the first phase and need to be passed into the second. There are alternate ways of passing environment variables from one image to the next, but I think in this case where the environment variables are sometimes passed in as an ARG and sometimes set from a command the only way of persisting them that I found to work was saving them to a file.

Reply:
Your logic makes sense @JoukoVirtanen. However, I do think that having multiple ways of instantiating these values introduces the potential for errors. For example, we won't log or exit here if the files are not found and the env vars are not already defined. 

Perhaps we should use the same technique as [Dockerfile.envsubst](https://github.com/stackrox/stackrox/blob/master/image/rhel/Dockerfile.envsubst) in stackrox and generate the Dockerfile with the correct env vars injected.